### PR TITLE
chore: [sc-1877] Cleanup Commands Interface

### PR DESCRIPTION
**Pull request by mjr-blockjoy**
proto cleanup according to BB Meeting
Story details: https://app.shortcut.com/blockjoy/story/1877

### CODE REVIEW

File: blockjoy/v1/command.proto [hunk 1]
```diff
@@ -139,10 +103,9 @@ message NodeCreate {
...
-  bool self_update = 7;
-  repeated Parameter properties = 8;
-  repeated Rule rules = 9;
-  string network = 10;
+  repeated Parameter properties = 7;
+  repeated Rule rules = 8;
+  string network = 9;
 }
 
 message NodeDelete {
```
Comment (chrisabruce) [inline blockjoy/v1/command.proto:111]: Don't we need the Node's id as well?

Comment (chrisabruce) [reply blockjoy/v1/command.proto:111]: Actually, this should be moved to `Node` proto as well?

**Review by chrisabruce — CHANGES_REQUESTED** [review]

File: blockjoy/v1/command.proto (hunk 1 above)
Comment (mjr-blockjoy) [reply blockjoy/v1/command.proto:111]: This one is part of  `NodeCommand` (line 95), which already has `node_id` and is sent from API to the Host

File: blockjoy/v1/command.proto [hunk 2]
```diff
@@ -23,56 +23,21 @@ message Command {
   }
 }
 
-// Service providing commands invoked by the UI
+// Service providing commands to be executed on Host
```
Comment (ThouCheese) [inline blockjoy/v1/command.proto:26]: on *a host

File: blockjoy/v1/command.proto [hunk 3]
```diff
@@ -23,56 +23,21 @@ message Command {
...
 service CommandService {
-  // Creates a new command in the database and sends it to the apropriate
-  // instance of blockvisor.
-  rpc Create(CommandServiceCreateRequest) returns
-    (CommandServiceCreateResponse);
-  // Retrieve a single command
-  rpc Get(CommandServiceGetRequest) returns (CommandServiceGetResponse);
-  // Update a single command status
+  // Update a single command status.
+  // This shall be used only by Host.
```
Comment (ThouCheese) [inline blockjoy/v1/command.proto:29]: *hosts

File: blockjoy/v1/command.proto [hunk 4]
```diff
@@ -23,56 +23,21 @@ message Command {
...
   rpc Update(CommandServiceUpdateRequest) returns
     (CommandServiceUpdateResponse);
   // Get pending commands for host. Pending is means that their execution has
   // not completed, be that successfully or unsuccessfully. This is defined as
   // commands that do not have an `exit_code`. To mark a command as no longer
   // pending, use the `Update` method and provide an `exit_code`.
+  // This can be used by anyone, but primarily by Host.
```
Comment (ThouCheese) [inline blockjoy/v1/command.proto:36]: *hosts

**Review by ThouCheese — APPROVED** [review]

**Pull request merged**